app_dart/lib/src/model/appengine/task.dart

```diff
@@ -132,6 +135,21 @@ class Task extends Model {
   @JsonKey(name: 'Reason')
   String reason;
 
+  /// The build ID of luci build: https://chromium.googlesource.com/infra/luci/luci-go/+/master/buildbucket/proto/build.proto#146
+  @IntProperty(propertyName: 'BuildID')
+  @JsonKey(name: 'BuildID')
```

Comment:
BuildNumber?

Reply:
Ha, thanks for catching this! just updated.